test_psa_target.py

```diff
@@ -345,6 +347,13 @@ def _get_parser():
         action="store_true",
     )
 
+    parser.add_argument(
+        "-s",
+        "--sync",
+        help="Enable waiting for sync from Greentea host",
+        action="store_true",
```

Comment:
So by default are we setting to 1?

Reply:
By default we set it to 0 if we don't pass this argument

Reply:
Now I've changed it to default to sync in `test_psa_target.py`.
In `mbed_app.json` it's still no sync by default, as people who build and run thing manually (without `test_psa_target.py`) expect test to run straightaway.

Reply:
Now I've changed it to default to sync in `test_psa_target.py`.
In `mbed_app.json` it's still no sync by default, as people who build and run thing manually (without `test_psa_target.py`) expect test to run straightaway.

Reply:
I think we should match `test_psa_target.py` and `mbed_app.json`, so there is no confusion

Reply:
Okay...

Reply:
Done